docs/development.md

```diff
@@ -92,27 +92,50 @@ list so that they will see CTRs as they come through. If another
 committer responds with a -1, the commit should be rolled back and the
 formal RTC process should be followed.
 
+### Enable Automatic Backporting
+
+Before merging the pull request, it should be decided whether the contribution should be backported to other release
+branches.
+This should be the case for most non-breaking changes.
+If the change applies to something that is not present at all on other still supported release branches, like updating
+a dependency that was only introduced on `master` or code that was added / heavily modified on `master`, then
+backporting cannot work or would require too much effort.
+Otherwise, the appropriate labels for backporting should be added before the pull requests gets merged as that allows
+the backporting action to automatically handle the backporting.
+
 ### Merging of Pull Requests
 
 A pull request is ready to be merged when it has been approved (see
-[Review-Then-Commit (RTC)](#review-then-commit)). It can either be
-merged manually with `git` commands or through the GitHub UI with the
-*Merge pull request* button. If the pull request targets a release
-branch that is upstream of other release branches (e.g., `v0.2` is
-upstream of `master`), then it is important to make sure that the change
-also lands in the downstream release branches. This is the
-responsibility of the committer who merges the pull request. Therefore,
-merge the pull request first into its target branch and then merge that
-branch (e.g., `v0.2`) into the next downstream branch (e.g., `v0.3`), and
-so on until you merge the last release branch into `master`. Afterwards
-push all release branches, ideally with an atomic commit:
+[Review-Then-Commit (RTC)](#review-then-commit-rtc)).
+It can either be merged manually with `git` commands or through the GitHub UI.
+
+For pull requests that should be backported (see [Enable Automatic Backporting](#enable-automatic-backporting)), the
+backporting action should create a pull request to backport the changes to other release branches after the pull
+request has been merged.
+The committer who merges the pull request should ensure that this automatic backporting succeeds.
+It may fail in case of merge conflicts.
+In that case the backporting needs to be [performed manually](#manual-backporting).
+
+### Manual Backporting
```

Comment:
I just tried out this tool and I kind of feel this is even more cumbersome than doing cherry-pick on my own. For example, it checks out a git repo under `.backport/` folder and I have to fix the conflicts there. I would prefer doing so manually on my working directory.

Reply:
I personally like the tool because I don't have to think much about what to do exactly, I only need to resolve the merge conflicts which is the only work that really needs to be done manually. I think is also less error prone than manual cherry picking, especially if there are multiple commits in the PR. But I have to say that I never really do cherry picking manually so I don't have much experience there.
But how about if we just add an additional sentence here that mentions that manual cherry picking can also be done instead of using the tool? Then people can decide on their own which approach they want to use.

Reply:
Sounds good!